core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala

```diff
@@ -68,21 +74,23 @@ class HttpMethodRepoDAO(agoraConfig: MethodRepoConfig[Agora.type], dockstoreConf
             tool: GA4GHTool <- maybeTool
           } yield WdlUrl(tool.url) // We submit the Github URL to Cromwell so that relative imports work.
         }
+      case DockstoreToolsMethod(methodPath, methodVersion) => {
+        val url = s"${dockstoreServiceURL}/ga4gh/v1/tools/${URLEncoder.encode(methodPath, UTF_8.name)}/versions/${URLEncoder.encode(methodVersion, UTF_8.name)}/WDL/descriptor"
```

Comment:
Might be nice to stash this big URL in a method that mirrors `getDockstoreMethod` and `ga4ghDescriptorUrl` for consistency

Reply:
I thought about that, but since the value is only used here I came to the opinion that reducing the code complexity by removing a layer of indirection was a win, even if it was slightly less consistent with the existing case.

Thoughts? (What about inlining the other one as well?)

Reply:
My thinking with the indirection is that `ga4ghDescriptorUrl` is a natural member of the domain object `DockstoreMethod`, it can be used to dereference a method and it itself is constructed from other members of the same object. I would prefer we keep that pattern but don't feel _too_ strongly.

Reply:
Thanks for the input, I'll maintain the existing pattern.

I'd like to continue exploring this concept as we go forward, since I believe in general that structuring code according to its current usage patterns yields a smaller and simpler code base, which is more amenable to reading and changing.

Reply:
Sounds good, we don't want to end up with a `DockstoreMethodFactory` or any other excessive OO